### DISCUSSION TITLE
libbpf FAILED

### DISCUSSION THREAD

**Discussion by jamroks** [loxilb-io/loxilb#612] [General]
Hi,
i launch an x86_64 ubuntu instance on a public subnet inside an aws vpc. i used this command : 
```
docker run -u root --cap-add SYS_ADMIN --net=host --restart unless-stopped --privileged -dit -v /dev/log:/dev/log --name loxilb ghcr.io/loxilb-io/loxilb:latest --host=0.0.0.0
```

name -r 
```
6.5.0-1016-aws
```

i tail the logs to take a look at the bootstrap and i found this error , i don't really know what it is , but i was concern since it's about ebpf 
```
libbpf: failed to statfs /opt/loxilb/dp/bpf: No such file or directory
```
You can found an extract of the logs attached to this issue 
[loxilb.txt](https://github.com/loxilb-io/loxilb/files/14817619/loxilb.txt)

please can you help me understand this error and how to troubleshoot

**Comment by UltraInstinct14** [comment]
Thanks @jamroks for reporting this. It seems to be a harmless error seen when creating eBPF filesystem. Still, this log should only get logged when there is a real issue to avoid confusion. Turning this into an issue for tracking !

**Reply by jamroks** [reply]
Then i'm reassured, Thank you for the quick response and for taking the necessary step to track the unnecessary logging message. kudos to the loxilb team 🙏🏽
